examples/chip-tool/commands/session-management/CloseSessionCommand.h

```diff
@@ -43,16 +62,33 @@ class CloseSessionCommand : public CHIPCommand
     }
 
 private:
-    chip::NodeId mDestinationId;
     chip::Optional<uint16_t> mTimeoutSecs;
+    chip::Optional<bool> mEvictLocalSession;
 
     static void OnDeviceConnectedFn(void * context, chip::Messaging::ExchangeManager & exchangeMgr,
                                     const chip::SessionHandle & sessionHandle);
     static void OnDeviceConnectionFailureFn(void * context, const chip::ScopedNodeId & peerId, CHIP_ERROR error);
 
-    // Try to send the action CloseSession status report.
+    // Try to send the CloseSession status report.
     CHIP_ERROR CloseSession(chip::Messaging::ExchangeManager & exchangeMgr, const chip::SessionHandle & sessionHandle);
 
     chip::Callback::Callback<chip::OnDeviceConnected> mOnDeviceConnectedCallback;
     chip::Callback::Callback<chip::OnDeviceConnectionFailure> mOnDeviceConnectionFailureCallback;
 };
+
+class EvictLocalCASESessionsCommand : public detail::SessionManagementCommand
+{
+public:
+    EvictLocalCASESessionsCommand(CredentialIssuerCommands * credIssuerCommands) :
+        detail::SessionManagementCommand("expire-CASE-sessions", credIssuerCommands,
```

Comment:
```suggestion
        detail::SessionManagementCommand("expire-case-sessions", credIssuerCommands,
```
I think chip tool commands are usually all lower case even if they contain acronyms?